apertium-eng.eng.rlx

```diff
@@ -329,6 +344,33 @@ SELECT V IF (-1 (prn pers)) ;
 SELECT SUB:1 Be IF (1 Ing) (0/* Have) ;
 SELECT Ger IF (-1/* Be) (1 To) ;
 
+SELECT Adv IF (0 Pr) (-1 Vblex) ;
+#Example (on):
+#He talked on.
+
+SELECT Cnjadv IF (0 Pr) (1 Det) (2 N) ;
+#Example (for):
+#Please get fish for the cat.
+
+SELECT Adv IF (0 N) (1 Supadj) ;
+#Example (smallest)
+#Canada is the second largest country in the world.
+
+SELECT Cnjadv IF (-1 Cm) (0 Pr) (1 Det) (2 N) ;
+#Example (for):
+#The cat didn't know, for the coconut didn't see.
+
+SELECT Cnjadv IF (-1 Cm) (0 Pr) (1 PrnSubj) ;
+#Example (for):
+#I was happy, for I had just won the lottery.
```

Comment:
I wouldn't rely too much on punctuation; you can't be sure it's going to be used the same way in all text.

Reply:
Perhaps it would be better to make this a remove rule.  Just remove the `pr` reading of there's a `prnsubj` after the ambiguous token?

Reply:
How?

Reply:
`REMOVE Pr IF (0 Cnjadv) (1 PrnSubj) ;`